src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/StatesPaidStaffNotificationServiceTest.java

```diff
@@ -0,0 +1,112 @@
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailProperties;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.response.DefenceType;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimantResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.claimstore.documents.output.PDF.EXTENSION;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildResponseFileBaseName;
+
+public class StatesPaidStaffNotificationServiceTest extends MockSpringTest {
+    private static final byte[] PDF_CONTENT = {1, 2, 3, 4};
+    private static final String DEFENDANT_EMAIL = "defendant@mail.com";
+    private Claim claimWithFullDefenceResponse;
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
+    @Captor
+    private ArgumentCaptor<EmailData> emailDataArgument;
+
+    @Autowired
+    private StaffEmailProperties emailProperties;
+
+    @Autowired
+    private StatesPaidStaffNotificationService service;
+
+    @Before
+    public void beforeEachTest() {
+        claimWithFullDefenceResponse = SampleClaim.builder()
+            .withDefendantEmail(DEFENDANT_EMAIL)
+            .withResponse(
+                SampleResponse.FullDefence
+                    .builder()
+                    .withDefenceType(DefenceType.ALREADY_PAID)
+                    .withMediation(null)
+                    .build())
+            .withRespondedAt(LocalDateTime.now())
+            .withClaimantResponse(SampleClaimantResponse.validDefaultAcceptation())
+            .build();
+
+
+        when(pdfServiceClient.generateFromHtml(any(byte[].class), anyMap()))
+            .thenReturn(PDF_CONTENT);
+    }
+
+    @Test
+    public void shouldSendEmailToExpectedRecipient() {
+
+        service.notifyStaffClaimantResponseStatesPaidSubmittedFor(claimWithFullDefenceResponse);
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
+
+        assertThat(senderArgument.getValue()).isEqualTo(emailProperties.getSender());
+    }
+
+    @Test
+    public void shouldSendEmailWithExpectedContentStatesPaidByFullDefence() {
+
+        service.notifyStaffClaimantResponseStatesPaidSubmittedFor(claimWithFullDefenceResponse);
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
+
+        assertThat(emailDataArgument.getValue()
+            .getSubject()).startsWith("Civil Money Claim States Paid");
```

Comment:
I think we should check subject equals to rather than starts with. See example here:  https://github.com/hmcts/cmc-claim-store/blob/b3b6c8cf3b2f8054b81877ca5e1ce0e424e0e8ef/src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/InterlocutoryJudgmentStaffNotificationServiceTest.java#L70